atlascope/tests/test_dataloader.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from django.contrib.auth.models import User
+import pytest
+from rest_framework import serializers
+
+from atlascope.core import models as atlascope_models
+from atlascope.core.management.commands.populate import DATALOADER_DIR, MODEL_JSON_MAPPING
+from atlascope.core.rest.additional_serializers import UserSerializer
+
+
+@pytest.mark.django_db
+def test_serializers_valid():
+    model_serializers = {
+        clas.Meta.model: clas
+        for clas in atlascope_models.__all__
+        if issubclass(clas, serializers.ModelSerializer)
+    }
+    model_serializers[User] = UserSerializer
+
+    for model, filename in MODEL_JSON_MAPPING:
+        objects = json.load(open(DATALOADER_DIR + filename))
+        for data_obj in objects:
+            model_serializers[model](data=data_obj).is_valid()
```

Comment:
```suggestion
            assert model_serializers[model](data=data_obj).is_valid()
```

Otherwise, the test will still pass if `.is_valid() == False`.

Reply:
of course! my bad, it was one of those instances where you quickly read a sentence and assume a word is there

Reply:
```suggestion
            model_serializers[model](data=data_obj).is_valid(raise_exception=True)
```

Alternatively, raise an exception. This might provide more meaningful test failure messages.

Reply:
Ah ok, should we switch to that option?